firebase_admin/_messaging_encoder.py

```diff
@@ -324,7 +325,12 @@ def encode_android_notification(cls, notification):
 
         event_time = result.get('event_time')
         if event_time:
-            result['event_time'] = str(event_time.isoformat()) + 'Z'
+            if event_time.tzinfo is None:
```

Comment:
I don't think you have to do any of this. At least on my machine, following works as expected.

```
>>> import datetime
>>> d = datetime.datetime.now()
>>> d
datetime.datetime(2020, 2, 3, 13, 39, 32, 133863)
>>> d.astimezone(datetime.timezone.utc)
datetime.datetime(2020, 2, 3, 21, 39, 32, 133863, tzinfo=datetime.timezone.utc)
```

The first is in my local timezone (13:39 PST). The second is in UTC (21:39 UTC). Isn't that all we want?

Reply:
Thanks! You're right. I missed that `astimezone` uses the local time zone if the provided datetime object is naive. Will update the code.